modules/olcne/variables.tf

```diff
@@ -100,3 +100,9 @@ variable "container_registry_urls" {
   }
   type = map(string)
 }
+
+variable "deploy_nginx_software" {

```

Comment:
no need for description and default here. You've already set it in root variables.

Reply:
> > If the user doesn't install an Ingress Controller and set the ServiceType to NodePort as we are doing here, they won't be able to access their services. So I'm wondering what is the purpose of giving the choice to turn it off.
> 
> Other projects might perform their own nginx-ingress installation, I understand your default is to install this non-OLCNE component but it should be a user option.

Fair enough and you've left the default to true 👍 I'll merge after you make the changes.

Reply:
> no need for description and default here. You've already set them in root variables.

Removed